### PR TITLE
Improve mobile UI for files list 

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description
This improvement solves the problem that files and folders with a longer name
than 10 chars will cut off to 7 chars by hiding non-essential meta data.
This is unwieldy to navigate through files and folders.
As well the uploading indicator text on a folders table row will now be
displayed at the beginning to avoid UI glitches.

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

### Before
before

### After 
after

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by JanAckermann** [comment]
@pmaier1 cool with it?

File: apps/files/css/mobile.css [hunk 1]
```diff
@@ -70,4 +66,14 @@ table td.filename .nametext .innernametext {
 	display: block !important;
 }
 
+/* hide mobile table content*/
+table #headerSize, #headerDate, .filesize, .modified, .date {
+		display: none;
```
Comment (ChrisEdS) [inline apps/files/css/mobile.css:71]: Code is indented incorrectly?

**Review by ChrisEdS — APPROVED** [review]
Technically approved.

**Comment by phil-davis** [comment]
Thinking out loud - I wonder what we should do about testing that the webUI "behaves nicely" when being displayed on a smaller device/window? It would be possible to run some CI with a smaller-sized browser window, but we can also just leave it to manual smoke-test to be done during release QA.

**Comment by JanAckermann** [comment]
@phil-davis Do we have similar tests already, how much effort do you think is required for a full coverage?

**Comment by phil-davis** [comment]
> @phil-davis Do we have similar tests already, how much effort do you think is required for a full coverage?

In web we set an env var, for example, `"SCREEN_RESOLUTION": "375x812"` to test with a mobile-phone-sized screen.

Here in the "traditional" oC10 UI we can do similar. Code like this resizes the window:
`$this->getSession()->resizeWindow(375, 812, 'current');`

We can add some code to do that if `SCREEN_RESOLUTION` is set. And then run whatever subset of test scenarios we like in CI with whatever `SCREEN_RESOLUTION` settings.

The "problem" is that we keep adding more CI pipelines! So maybe we just do it for a set of "smokeTest" scenarios.

**Comment by JanAckermann** [comment]
@pmaier1 waiting for your go :+1:

**Comment by pmaier1** [comment]
> @pmaier1 waiting for your go

Fine for me :+1:

**Pull request merged**